app/templates/errors/400.html

```diff
@@ -0,0 +1,24 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Bad request - Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+<div class="error-page">
+  <div class="index-page grid-row">
+    <div class="column-two-thirds">
+      <header class="page-heading-smaller">
+        <h1>Sorry, there was a problem with your request</h1>
+      </header>
+      <p>
+        {% if error_message %}
+          {{ error_message }}
+        {% else %}
+          Please do not attempt the same request again.
```

Comment:
Assume there's only contrived ways for this to happen, but what comes to mind is multiple user concurrency, so when we add collaboration we could think about how to make the recovery more friendly (e.g. provide a link back to the project)

Reply:
This was mostly copied from the user-frontend, but yeah we should sort out the content eventually.

We should bring all the error message templates into the frontend-toolkit eventually. I started doing that but it seems like a non-trivial thing to link them all up from a central place, so shelved it for now. I dropped a ticket in the tech debt board a month or two ago about it anyway.

Reply:
Regarding contrived ways for this to happen: if you go back in the browser you can hit this page again, and in Chrome it automatically re-sends the request so it's not that difficult to conceivably hit this 400 as a fairly average user. But yeah we could probably have something nicer once collaboration comes in.